package.json

```diff
@@ -242,7 +242,7 @@
     "taskr": "1.1.0",
     "tree-kill": "1.2.2",
     "tsec": "0.2.1",
-    "turbo": "2.0.9",
+    "turbo": "2.1.2",
```

Comment:
This also needs bumping in our CI config to ensure we're in sync

https://github.com/vercel/next.js/blob/6a69c358ecfc6f293676b2110a88fe3004166ea8/.github/workflows/build_and_deploy.yml#L12
https://github.com/vercel/next.js/blob/6a69c358ecfc6f293676b2110a88fe3004166ea8/.github/workflows/build_and_test.yml#L11